controllers.js

```diff
@@ -17,8 +17,28 @@ exports.fetchTopics = (req, res, next) => {
     })
     .catch(next)
 };
+exports.fetchArticle = (req, res, next) => {
+  const { article_id } = req.params;
 
+  
+  if (isNaN(article_id)) {
+      return res.status(400).send({ message: 'Bad Request!' });
```

Comment:
a psql error will be thrown if id is not a number.   we want to catch this in our error handing middleware in our app.js.